### PR TITLE
Inner twitter api requests

### PR DESCRIPTION
**Pull request by nirdz**
Using inner api to get tweet page + formatting the tweet page to the same format as the official api

### CODE REVIEW

File: business_logic/participant/participant_manipulated_data/participantSpecifiedTwitterData.js [hunk 1]
```diff
@@ -1,52 +1,42 @@
 const twitterComm = require("../../twitter_communicator/twitterCommunicator")
-const manipulator = require("../manipulator/manipulator.js")
-const database = require("../../db/DBCommunicator.js");
+const twitterInnerApiUtils = require("../../twitter_communicator/twitter_internal_api/twitterInnerApiUtils")
 
 async function getUser(username){ // Later we will also send "praticipant" obj for manipulations.
```
Comment (talfrim) [inline business_logic/participant/participant_manipulated_data/participantSpecifiedTwitterData.js:4]: time to delete this comment?

Comment (nirdz) [reply business_logic/participant/participant_manipulated_data/participantSpecifiedTwitterData.js:4]: The one with "Later we will..."? Yea I'll delete it

File: business_logic/twitter_communicator/twitter_api_get/twitterApiGet.js [hunk 2]
```diff
@@ -40,10 +40,40 @@ async function verifyCredentials(T){
...
+async function getUserLikes(T, username){
+    const params = {
+        screen_name: username,
+        tweet_mode: "extended",
+        count: 40
+    }
+    return await sendGetRequestsWrapper(T, "favorites/list", params)
+}
+
+async function getUserFriends(T, username){
```
Comment (talfrim) [inline business_logic/twitter_communicator/twitter_api_get/twitterApiGet.js:57]: **very** pro looking

**Review by talfrim — APPROVED** [review]
bravo

**Pull request merged**